aws/Full-PAS-Deployment.json

```diff
@@ -1160,6 +1165,11 @@
                         "waitAfterCompletion": "0",
                         "ignoreErrors": "false"
                      },
+                     "03-CheckVaultServiceStatus": {
```

Comment:
In DR PrivateArk Server should be stopped

Reply:
Done